DOCS-1548 - Mobot, Example Prompts, AI/ML FAQ | GA docs (#6897)
SumoLogic/sumologic-documentation: docs/search/mobot/index.md

---
slug: /search/mobot
title: Sumo Logic Mobot
sidebar_label: Mobot ✨
description: Troubleshoot logs, create monitors and dashboards, and learn the platform using plain-language questions with Mobot, Sumo Logic's AI assistant.
keywords:
  - mobot
  - dojo ai
  - ai assistant
  - agent
  - log analysis
  - log troubleshooting
  - unstructured logs
  - soc analyst
---

import useBaseUrl from '@docusaurus/useBaseUrl';
import SumoAcademy from '../../reuse/sumo-logic-academy.md';
import Iframe from 'react-iframe';

<img src={useBaseUrl('img/icons/operations/mobot.png')} alt="Search icon" width="35"/>

Mobot is Sumo Logic's AI-powered conversational assistant for security analysts, on-call engineers, administrators, and other Sumo Logic users. Ask questions in plain language to investigate log data or learn how to use the platform without selecting an agent or writing queries from scratch.

<img src={useBaseUrl('img/search/mobot/general-tab.png')} alt="Mobot welcome screen with example prompts grouped by General, Security, Observability, and Administration" style={{border: '1px solid gray'}} width="700" />

> In a single conversation, Mobot determines whether you have a log data question or a how-to question and responds accordingly. For log data questions, it identifies relevant sources, correlates information across logs, and returns inline results with anomaly callouts and suggested next steps. For how-to questions, it provides structured answers and reference links from Sumo Logic documentation. Conversation context lets you refine, pivot, and dig deeper without starting over.

Reviewer: ```suggestion
In a single conversation, Mobot determines whether you have a log data or a how-to question and responds accordingly. For log data questions, it identifies relevant sources, correlates information across logs, and returns inline results with anomaly callouts and suggested next steps. For how-to questions, it provides structured answers and reference links from Sumo Logic documentation. Conversation context lets you refine, pivot, and dig deeper without starting over.
```

* **Content creation and management**. Create and manage monitors, dashboards, and playbooks through conversation, with a human in the loop before anything is deployed or activated. See [Create and manage content](#create-and-manage-content).
* **Cloud SIEM investigations**. On a Cloud SIEM insight, click **Ask Mobot** to continue a [SOC Analyst Agent](/docs/cse/get-started-with-cloud-siem/soc-analyst-agent) investigation in Mobot with the insight's evidence-backed verdict and context loaded. You start this from the insight, not from Mobot directly.
* **Conversation history**. Saved conversations let you resume, revisit, or branch previous investigations.

## At a glance

- **Response time**. Typically under 2 seconds for most queries. See [response time FAQ](#what-is-the-typical-mobot-response-time) for details.
- **Compatible log types**. Structured, semi-structured, and unstructured logs. Unstructured logs already used in dashboards do not require Field Extraction Rules. See [Compatible log formats](#compatible-log-formats) for details.
- **AI provider**. Amazon Bedrock (no customer data used for training). See [Security and compliance](#security-and-compliance) for details.

> :::training Sumo Logic Academy

Reviewer: Can we comment out the self-paced until next week when training courses are ready? I updated the placeholder link. The live training entry is live and works fine.

<SumoAcademy/>

> * **Self-paced**. [Mobot Essentials Self Paced](https://learn.sumologic.com/mobot-101).

Reviewer: ```suggestion
* **Self-paced**. [Hands on with Mobot and Dojo AI](https://learn.sumologic.com/hands-on-with-mobot-and-dojo-ai).
```

> * **Instructor-led virtual classes**. [Workshops: Mobot Essentials](https://www.sumologic.com/learn/training?_workshops=mobot-essentials#section-2).

Reviewer: ```suggestion
* **Instructor-led virtual classes**. [Workshops: Hands on with Mobot and Dojo AI](https://www.sumologic.com/learn/training?_workshops=hands-on-mobot-dojo-ai#section-2).
```

:::

:::training Micro Lesson

Watch this micro lesson to learn what Mobot is, its key capabilities, and how its AI-powered, unified conversational interface accelerates your investigations.

> <Iframe url="https://fast.wistia.net/embed/iframe/t187881hs5op2qi?web_component=true&seo=true&videoFoam=false"

Reviewer: ```suggestion
<Iframe url="https://fast.wistia.net/embed/iframe/8jgl5h69np?web_component=true&seo=true&videoFoam=false"
```

Reviewer: original iframe url for this video was invalid

At GA, Mobot shares a single **AI features** toggle with Parse Assist and the SOC Analyst Agent — turning it off disables all three together. Independent per-feature toggles, starting with the SOC Analyst Agent, are planned for a future release.

Parent and child orgs have AI features enabled by default. A parent org administrator can toggle AI features for the parent org and for its child orgs. Child org administrators cannot toggle AI features for their own org or for other child orgs. MSSP parent orgs do not see this toggle for their own org.

<img src={useBaseUrl('img/search/mobot/feature-management.png')} alt="Feature Management page showing the AI features and MCP Server access toggles" style={{border: '1px solid gray'}} width="800" />

If you previously opted out of Mobot, you'll need to opt back in from this page to regain access at GA, or contact your account team. Trial accounts do not have Mobot enabled by default.

For help, contact our [Support team](https://support.sumologic.com/support/s/).

## Additional resources

* [AI and Machine Learning with Sumo Logic](/docs/get-started/ai-machine-learning)

> * [Mobot Essentials Self Paced Course](https://learn.sumologic.com/mobot-101).

Reviewer: ```suggestion
* [Hands on with Mobot and Dojo AI - Self Paced Training](https://learn.sumologic.com/hands-on-with-mobot-and-dojo-ai).
```

> * [Mobot Instructor-Led Virtual Workshops](https://www.sumologic.com/learn/training?_workshops=mobot-essentials#section-2).

Reviewer: ```suggestion
* [Mobot Instructor-Led Virtual Workshops](https://www.sumologic.com/learn/training?_workshops=hands-on-mobot-dojo-ai#section-2).
```

### Open in Log Search

Open your query in [Log Search](/docs/search) to access full search functionality, take action, or save the search for later. There are two ways to do this:

From your conversation, click on a results bubble.<br/><img src={useBaseUrl('img/search/mobot/open-in-log-search-view-results-mobotv2.png')} alt="Mobot conversation bubble showing clickable results to open in Log Search" style={{border: '1px solid gray'}} width="600" />

Or, from the query section, click **Open in Log Search**.<br/><img src={useBaseUrl('img/search/mobot/open-in-log-search.png')} alt="Open in Log Search icon button in Mobot query section" style={{border: '1px solid gray'}} width="600" />

### Audit Mobot queries

:::info Prerequisites
* You need to be an account administrator.
* The [search audit index](/docs/manage/security/audit-indexes/search-audit-index) needs to be enabled for your organization.
:::

Administrators can audit queries generated by Mobot using the search audit index.

To view Mobot queries:
1. Open **Log Search**.
1. Use the following query:
   ```sumo
   _view=sumologic_search_usage_per_query
   | where query_type in ("Query Agent")
   | count user_name, query
   ```

>    <!-- TODO (DOCS-1548): Confirm with engineering whether the query_type field value was updated to "Log Analysis Agent" for the rename, or still logs "Query Agent" on the backend. Update this filter value if it changed. -->

Author: @tp-sl

:::

### Role-based access control

Role-based access control (RBAC) is not supported for contextual suggestions and autocompletions. A user blocked by [log search RBAC](/docs/manage/users-roles/roles/construct-search-filter-for-role/) may see suggestions for unpermitted source expressions, but those searches will not execute.

### Search behavior and data tier access

Mobot follows the same search behavior as standard log search and respects your account's data configuration.

* **Flex pricing**. If you are on [Flex pricing](/docs/manage/partitions/flex), all data is stored in a single intelligent layer and pricing is based on the volume of data scanned.
* **Tiered pricing**. If you are on a [tiered pricing](/docs/manage/partitions/data-tiers/searching-data-tiers/) plan, Mobot searches across continuous data tiers only, unless you specify otherwise. To query a specific tier, include the `_dataTier` field in your prompt. For example:
   ```sumo
   _dataTier=Infrequent
   ```

> <!-- TODO (DOCS-1548): Open question raised internally — Mobot may search partitions marked 'excluded' from default scope, as well as 'infrequent' partitions, when a user hasn't explicitly scoped their prompt. Unconfirmed whether this is intended behavior or whether customers can limit which partitions Mobot can scan. A teammate noted similar language already exists for MCP & Flex docs, but was unsure if an equivalent note exists for Mobot. Confirm with product and document here if it's a real behavior/limitation. -->

Author: @tp-sl
